tests/testthat/test-utils.R

```diff
@@ -251,8 +251,8 @@ test_that("guess_period works", {
     weekly_posixcts
   )
   # On POSIXlts:
-  daily_posixlts <- as.POSIXlt(daily_dates, tz = "ET") + 3600
-  weekly_posixlts <- as.POSIXlt(weekly_dates, tz = "ET") + 3600
+  daily_posixlts <- as.POSIXlt(daily_dates, tz = "UTC") + 3600
+  weekly_posixlts <- as.POSIXlt(weekly_dates, tz = "UTC") + 3600
```

Comment:
```suggestion
  daily_posixlts <- as.POSIXlt(daily_dates, tz = "US/Aleutian") + 3600
  weekly_posixlts <- as.POSIXlt(weekly_dates, tz = "US/Aleutian") + 3600
```
these actually properly use an OlsonNames tz (... were we hitting the unrecognized-tz-treated-as-UTC case??  ugh).  US/Aleutian seems like a decent "bad" case: not in any dev timezones, and has DST.  Hopefully in future tests we might write it could catch stuff.  Though if we really are doing datetime stuff we'd want also no-provided-tz tests, etc.

Reply:
My local checks were producing a warning on the previous "ET". Don't have a preference for which tz we use.